conans/client/conan_api.py

```diff
@@ -827,6 +827,32 @@ def remote_update_ref(self, reference, remote_name):
         reference = ConanFileReference.loads(str(reference))
         return self._registry.refs.update(reference, remote_name)
 
+    @api_method
+    def remote_list_pref(self, reference):
+        reference = ConanFileReference.loads(str(reference))
```

Comment:
Add validation, it is an user input:  ``ConanFileReference.loads(str(reference), validate=True)``.... in the future that argument should be ``False`` by default.